src/VisionModel/googlenet.jl

```diff
@@ -0,0 +1,42 @@
+export googlenet
+
+function conv_block(inplanes, outplanes, stride, pad, kernel_size)
+  conv_layer = Chain(Conv(kernel_size, inplanes => outplanes, stride = stride, pad = pad),
+                     BatchNorm(outplanes, relu))
+  return conv_layer
+end
+
+function inception_block(inplanes, out_1x1, red_3x3, out_3x3, red_5x5, out_5x5 out_1x1pool)
+  inception_layer = Chain(conv_block(inplanes, out_1x1, kernel_size=(1,1)),
+                          conv_block(inplanes, red_3x3, kernel_size=(1,1))
+                          conv_block(red_3x3, out_3x3, pad=1, kernel_size=(3,3)),
+                          conv_block(inplanes, red_5x5, kernel_size=(1,1)),
+                          conv_block(red_5x5, out_5x5, pad=2, kernel_size=(5,5)),
+                          MaxPool((3, 3), stride=1, pad=1),
+                          conv_block(inplanes, out_1x1pool, kernel_size=(1,1)))
+  return inception_layer
+end
+
+function googlenet(inplanes, )
+  layers = Chain(conv_block(inplanes=3, outplanes=64, stride=2, pad=3, kernel_size=(7,7)),
+                 MaxPool((3,3), stride=2, pad=1),
+                 conv_block(inplanes=64, outplanes=192, stride=1, pad=1, kernel_size=(3,3)),
```

Comment:
This should be a 1x1 "reduction" convolution followed by a 3x3 convolution. See Table 1 in the paper (screenshot below). As you can see, the 112x112x64 input is reduced to 56x56x64 by a 1x1 reduction (in the accompanying paragraph they explain that "#3x3 reduce" corresponds the number of filters in 1x1 conv), then the output channels are expanded to 56x56x192 by a 3x3 conv. You can see the same in the screenshot below of Fig. 3 from the paper.
<img width="777" alt="Screen Shot 2020-07-29 at 9 21 31 AM" src="https://user-images.githubusercontent.com/4811350/88812106-f8e30980-d17c-11ea-8553-d0355349ee39.png">

<img width="77" alt="Screen Shot 2020-07-29 at 9 25 08 AM" src="https://user-images.githubusercontent.com/4811350/88812504-67c06280-d17d-11ea-8f14-ff04de500299.png">

Reply:
yes, my bad!